python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -394,6 +394,21 @@ def to_default_asset_events(
 DbtDagsterEventType = Union[Output, AssetMaterialization, AssetCheckResult, AssetObservation]
 
 
+class RelationKey(NamedTuple):
```

Comment:
Any reason why we don't just use `BaseRelation` as the key?

Reply:
Slight trepidation over using class we don't control as a key even if hashable since it might change under us - `RelationKey` is a little more minimal too

Reply:
makes sense